server/vespa/reindex.sh

```
@@ -0,0 +1,80 @@
+  echo "Error starting reindexing: $error_message"
+  exit 1
+fi
+
+
+success_message=$(echo "$response")
+echo "Starting reindex for document type: $success_message"
+
+vespa prepare
+vespa activate
+
+IFS=',' read -ra docTypes <<< "$documentTypes"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, read -a is undefined.

(SC3045)